### DISCUSSION TITLE
[Feature] Album - delete multiple

### DISCUSSION THREAD

**Discussion by blublub** [immich-app/immich#10779] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi

Since we can select multiple images for delete I would think it is logical to have the same option for albums, at least in the "list view" with a "tickbox on the left" that would be useful I think

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8543
